paying_for_college/disclosures/scripts/purge_objects.py

```diff
@@ -0,0 +1,21 @@
+from paying_for_college.models import Notification, Program
+
+error_msg = "Only Notification and Program objects may be purged"
```

Comment:
It would be helpful if we mention using `notifications` and `programs` only as argument.